zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -343,6 +349,26 @@ public void parseProperties(Properties zkProp) throws IOException, ConfigExcepti
                 snapRetainCount = Integer.parseInt(value);
             } else if (key.equals("autopurge.purgeInterval")) {
                 purgeInterval = Integer.parseInt(value);
+            } else if (key.equals(BackupSystemProperty.BACKUP_ENABLED)) {
+                backupConfigBuilder.setEnabled(Boolean.parseBoolean(value));
+            } else if (key.equals(BackupSystemProperty.BACKUP_STATUS_DIR)) {
+                backupConfigBuilder.setStatusDir(vff.create(value));
+            } else if (key.equals(BackupSystemProperty.BACKUP_TMP_DIR)) {
+                backupConfigBuilder.setTmpDir(vff.create(value));
+            } else if (key.equals(BackupSystemProperty.BACKUP_INTERVAL_MINUTES)) {
+                backupConfigBuilder.setBackupIntervalInMinutes(Long.parseLong(value));
```

Comment:
Should use parseInt() here since BACKUP_INTERVAL_MINUTES is an int

Reply:
Good catch